includes/REST_API/Stories_Page_Template_Controller.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Class Stories_Page_Template_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use stdClass;
+use WP_Error;
+use WP_Post;
+use WP_REST_Posts_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+/**
+ * Class Stories_Page_Template_Controller
+ *
+ * @package Google\Web_Stories\REST_API
+ */
+class Stories_Page_Template_Controller extends WP_REST_Posts_Controller {
```

Comment:
Nit: remove the `Stories_` prefix

Reply:
All of our other controllers are named like this. 
<img width="279" alt="Screenshot 2021-04-08 at 16 16 07" src="https://user-images.githubusercontent.com/237508/114052217-c3eb9280-9885-11eb-9b43-5f00c6961668.png">